server/actions/user-actions.js

```diff
@@ -45,7 +45,10 @@ class UserActions {
             if (err) {
                 res.status(400).send(err);
             } else if (users == '') {
-                res.status(404).send("No users with role '" + req.body.role + "' not found");
+                res.status(200).send({
+                    statusCode: 404,
```

Comment:
Change the statusCode to: `error: not_found` or just remove it completely, doesn't make sense to have 2 status codes